DBMS module/System_requirement_analysis.md

```diff
@@ -5,7 +5,7 @@ model is used to form commonality relationships between ER models as a basis for
 integration.
 
 <p align="center">
-<img width="700" height="535" src="images/ER-model.png"> 
+<img width="700" height="535" src="../images/ER-model.png"> 
```

Comment:
@Antsthebul  Tomy I don't think  two dots are necessary. It will also work fine with by removing the dots and keeping only forward slash